playground/src/MonacoWrapper.tsx

```diff
@@ -167,6 +190,67 @@ export class MonacoWrapper extends React.Component<IMonacoWrapperProps, IMonacoW
     }
   }
 
+  private _updateDecorations(newDecorations: ISyntaxDecoration[]): void {
```

Comment:
>_updateDecorations [](start = 10, length = 18)

Maybe `_applySyntaxHighlighting()` or `_updateSyntaxHighlighting()`?

"Syntax highlighting" is the industry term for what we're doing.  Decorators are the technology used to implement it.